ci-operator/config/stackrox/collector/stackrox-collector-ROX-11211-collector-ci-migration-images.yaml

```diff
@@ -13,6 +16,27 @@ images:
 - dockerfile_path: .openshift-ci/build/Dockerfile.build_collector
   from: collector-builder
   to: collector
+- dockerfile_path: .openshift-ci/build/Dockerfile.drivers
+  from: collector
+  to: collector-drivers
+- dockerfile_path: .openshift-ci/build/Dockerfile.create_bundle
+  from: collector-drivers
+  to: collector-bundle
+- build_args:
+  - name: ROOT_DIR
+    value: /go/src/github.com/stackrox/collector/collector/container/rhel
+  dockerfile_path: collector/container/Dockerfile.gen
+  from: ubi-minimal
+  context_dir: collector/container/rhel
+  inputs:
+    collector-bundle:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/stackrox/collector/.
+  to: collector-full
+promotion:
```

Comment:
What exactly is this for? I was thinking that image will be published via a separate test step?